### ISSUE TITLE
Putting water with 'supplementaries:faucet' in a cauldron completely removes impurities from water

### ISSUE THREAD

**Issue by fkyah3** [ghen-git/Thirst-Mod#108]
No matter what kind of container is used, as long as the water enters the cauldron through the faucet, the water will become the vanilla water without showing whether it contains impurities. There will be no such problem when entering the 'create:basin'

**Comment by mlus-asuka** [comment]
yes,cauldron is different from mod containers. we need to pay lots of efforts to make it compatible.
